### PR TITLE
Task: Add device telemetry

### LINKED ISSUE #2001: Add telemetry on device characteristics
In order to get responsiveness right and provide a good experience for our customers, we first need to understand what devices and screen resolutions are most used. 
The goal here is to add telemetry to capture screen resolution and other device characteristics that will help us understand usage patterns and build appropriately.

### PR DESCRIPTION
**Pull request by MaryannGitonga**
## Overview

Closes #2001 

### Demo

Optional. Screenshots, `curl` examples, etc.

### How to test this PR
-

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,8 +14,9 @@
                 "@microsoft/applicationinsights-react-js": "3.3.6",
                 "@microsoft/applicationinsights-web": "2.8.6",
                 "@microsoft/microsoft-graph-client": "3.0.2",
+                "@microsoft/oteljs": "file:packages/oteljs-4.8.18.tgz",
                 "@monaco-editor/react": "4.4.5",
-                "@ms-ofb/officebrowserfeedbacknpm": "file:packages/officebrowserfeedbacknpm-1.6.6.tgz",
+                "@ms-ofb/officebrowserfeedbacknpm": "file:packages/officebrowserfeedbacknpm-2.0.0.tgz",
```
Comment (Onokaev) [inline package.json:19]: Is this change related to this PR?

File: package.json [hunk 2]
```diff
@@ -32,6 +34,7 @@
                 "file-loader": "6.2.0",
                 "fork-ts-checker-webpack-plugin": "7.2.13",
                 "fs-extra": "10.1.0",
+                "gatecrash": "file:packages/gatecrash-1.3.12.tgz",
```
Comment (Onokaev) [inline package.json:37]: what does this package do?

File: package.json (hunk 1 above)
Comment (MaryannGitonga) [reply package.json:19]: Oh no! It doesn't. My bad. Let me remove that.

File: package.json (hunk 2 above)
Comment (MaryannGitonga) [reply package.json:37]: That as well isn't related to the PR. Sorry!
